spring-cloud-dataflow-module-deployer-spi/src/main/java/org/springframework/cloud/dataflow/module/deployer/ModuleDeployer.java

```diff
@@ -38,6 +39,8 @@
 
 	public static final String JMX_DEFAULT_DOMAIN_KEY  = "spring.jmx.default-domain";
 
+	public static final String MODULE_DEPLOYMENT_GROUP_ID = "spring_cloud_dataflow.module_deployment_group_id";
```

Comment:
should this use dots and hyphens rather than underscores? (e.g. `spring.cloud.dataflow.module-deployment-group-id`),... and is "dataflow" sufficient for the prefix?